Structure_Engine/Query/OutlineElements1D.cs

```diff
@@ -0,0 +1,58 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2023, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BH.oM.Base.Attributes;
+using BH.oM.Dimensional;
+using BH.oM.Geometry;
+using BH.oM.Structure.Elements;
+using BH.Engine.Geometry;
+
+
+namespace BH.Engine.Structure
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Returns the outline 1D elements of a PadFoundation.")]
+        [Input("padFoundation", "The PadFoundation to find the outline elements for.")]
+        [Output("outlineElements", "A collection of outline 1D elements.")]
+        public static List<IElement1D> OutlineElements1D(this PadFoundation padFoundation)
+        {
+            if (padFoundation.IsNull())
+                return new List<IElement1D>();
```

Comment:
I would rather return `null` here - how can a `null` object return a collection? As long as it is not a well established pattern (however arguable it feels to me now)